isis/src/tgo/tsts/stitchUnstitch/Makefile

```diff
@@ -0,0 +1,79 @@
+APPNAME =
+
+include $(ISISROOT)/make/isismake.tsts
+
+commands:
+	tgocassis2isis $(TSTARGS) \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-PAN-00005-B1.xml \
+            to=$(OUTPUT)/panframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS) \
+            from=$(OUTPUT)/panframelet.cub \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS) \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-RED-01005-B1.xml \
+            to=$(OUTPUT)/redframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS) \
+            from=$(OUTPUT)/redframelet.cub \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS) \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-BLU-03005-B1.xml \
+            to=$(OUTPUT)/bluframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS) \
+            from=$(OUTPUT)/bluframelet.cub \
+            ckp=t spkp=t > /dev/null;
+
+	tgocassis2isis $(TSTARGS) \
+            from=$(INPUT)/CAS-MCO-2016-11-26T22.50.27.381-NIR-02005-B1.xml \
+            to=$(OUTPUT)/nirframelet.cub > /dev/null;
+
+	spiceinit $(TSTARGS) \
+            from=$(OUTPUT)/nirframelet.cub \
+            ckp=t spkp=t > /dev/null;
+
+	$(LS) $(OUTPUT)/*cub > $(OUTPUT)/cubelist.lis;
+	tgocassisstitch $(TSTARGS) \
+            fromlist=$(OUTPUT)/cubelist.lis \
+            outputprefix=$(OUTPUT)/"stitched" > /dev/null;
+
+	tgocassisunstitch $(TSTARGS) \
+            from=$(OUTPUT)/stitched-2016-11-26T22:50:27.381.cub \
+            outputprefix=$(OUTPUT)/"unstitched" > /dev/null;
+
+	cubediff $(TSTARGS) \
+            from=$(OUTPUT)/nirframelet.cub \
+            from2=$(OUTPUT)/unstitched_NIR.cub \
+            to=$(OUTPUT)/nirdiff > /dev/null;
```

Comment:
Add .pvl extension to the output diff files to ensure that pvldiff is used to check them.

Reply:
the .pvl extension is automatically added to the output file.

Reply:
This passes its cat test on mac as well.